plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/testing/TpchHudiTablesInitializer.java

```diff
@@ -334,24 +334,22 @@ private enum TpchColumnTypeAdapter
 
         static TpchColumnTypeAdapter of(TpchColumnType columnType)
         {
-            if (columnType == TpchColumnTypes.INTEGER) {
```

Comment:
In fact this is correct. `TpchColumnTypes` guarantees only single instance of INTEGER type.